### PR TITLE
[enhance](Recycler) Make sheme of request to object storage a config

### PR DESCRIPTION
**Pull request by ByteYue**

### CODE REVIEW

**Comment by ByteYue** [comment]
run buildall

File: cloud/src/recycler/s3_accessor.cpp [hunk 1]
```diff
@@ -246,6 +247,9 @@ int S3Accessor::init() {
         Aws::Client::ClientConfiguration aws_config;
         aws_config.endpointOverride = conf_.endpoint;
         aws_config.region = conf_.region;
+        if (config::s3_client_http_scheme == "HTTP") {
```
Comment (gavinchou) [inline cloud/src/recycler/s3_accessor.cpp:250]: this is incorrect compared to the config.h validation.
make it ignore case

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Comment by ByteYue** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**